cmd/hope/utils/nodes.go

```diff
@@ -15,6 +15,18 @@ import (
 	"github.com/Eagerod/hope/pkg/kubeutil"
 )
 
+type ErrNodeNotFound struct {
+	node string
+}
+
+func NewErrNodeNotFound(node string) error {
+	return ErrNodeNotFound{node}
```

Comment:
`NewXyz` should return a pointer, and the function below should have a pointer receiver.